### PR TITLE
Gary/add login prompt

### PR DESCRIPTION
**Pull request by Gazwai**
- Added a Login prompt snippet which can be rendered when the user is not logged in

### CODE REVIEW

File: snippets/easy_points_login_prompt.liquid [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{%- comment -%} START EASYPOINTS - DO NOT MODIFY! 変更しないでください! INFO @ https://bit.ly/2Dn7ESM {%- endcomment -%}
+{%- assign shop_ep_metafield = shop.metafields.loyalty.easy_points_attributes.value -%}
+  {%- unless shop_ep_metafield.stealth_mode -%}
+    {%- unless customer -%}
+      <small class="ep-disabled-text">ポイントのご利用には<a href="/account/login">ログイン</a>していただく必要がございます。</small>
```
Comment (NexPB) [inline snippets/easy_points_login_prompt.liquid:5]: - For routing use https://shopify.dev/api/liquid/objects#routes
- Text should be translated. But translations have to be added in the app when we insert our main assets.

File: assets/easy_points_integration.css [hunk 2]
```diff
@@ -171,6 +171,10 @@
   font-size: 95%;
 }
 
+.ep-disabled-text a {
+  opacity: 0.7;
```
Comment (NexPB) [inline assets/easy_points_integration.css:175]: Maybe we should just use `text-decoration: underline;`

File: snippets/easy_points_login_prompt.liquid [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{%- comment -%} START EASYPOINTS - DO NOT MODIFY! 変更しないでください! INFO @ https://bit.ly/2Dn7ESM {%- endcomment -%}
+{%- assign shop_ep_metafield = shop.metafields.loyalty.easy_points_attributes.value -%}
+  {%- unless shop_ep_metafield.stealth_mode -%}
+    {%- unless customer -%}
```
Comment (NexPB) [inline snippets/easy_points_login_prompt.liquid:4]: Double unless blocks use `and`

**Review by NexPB — CHANGES_REQUESTED** [review]

File: assets/easy_points_integration.css (hunk 2 above)
Comment (Gazwai) [reply assets/easy_points_integration.css]: Changed to underline

File: snippets/easy_points_login_prompt.liquid (hunk 3 above)
Comment (Gazwai) [reply snippets/easy_points_login_prompt.liquid]: Used  ```or``` for the the double unless

File: snippets/easy_points_login_prompt.liquid (hunk 1 above)
Comment (Gazwai) [reply snippets/easy_points_login_prompt.liquid]: I changed the routing to use the liquid routes

**Comment by Gazwai** [comment]
Tested on dev theme with ja.json directly manipulated

Screen Shot 2022-09-09 at 17 55 12
Screen Shot 2022-09-09 at 17 55 16

File: snippets/easy_points_login_prompt.liquid [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{%- comment -%} START EASYPOINTS - DO NOT MODIFY! 変更しないでください! INFO @ https://bit.ly/2Dn7ESM {%- endcomment -%}
+{%- assign shop_ep_metafield = shop.metafields.loyalty.easy_points_attributes.value -%}
+{%- unless shop_ep_metafield.stealth_mode or customer -%}
+  <a
+    href="{{ routes.account_login_url }}"
+    class="ep-disabled-text"
```
Comment (NexPB) [inline snippets/easy_points_login_prompt.liquid:6]: I don't like the naming of this css class here, it's not _disabled_.

**Review by NexPB — CHANGES_REQUESTED** [review]

File: snippets/easy_points_login_prompt.liquid (hunk 4 above)
Comment (Gazwai) [reply snippets/easy_points_login_prompt.liquid:6]: Very true, I'll modify that

**Review by NexPB — APPROVED** [review]

**Pull request merged**